_dev/apps/ui/src/store/modules/accounts/actions.ts

```diff
@@ -69,6 +70,13 @@ export default {
     dispatch(ActionsTypes.SEND_GMC_INFORMATION_TO_SHOP, {
       id: selectedAccount.id,
     });
+    const getModulesStatus = await dispatch(`app/${ActionsTypesApp.GET_MODULES_VERSIONS}`, 'psxmarketingwithgoogle', {root: true});
+
+    if (getModulesStatus?.hooks) {
+      Object.keys(getModulesStatus.hooks)
+        .map((item) => dispatch(`app/${ActionsTypesApp.TRIGGER_REGISTER_HOOK}`, item, {root: true}));
+    }
```

Comment:
Any chance to filter the hooks that are already registered?

Reply:
Sure, I made a modification to send only hook not registered